### ISSUE TITLE
Why action on handle submit do not validate each time?

### ISSUE THREAD

**Issue by alex-shamshurin** [erikras/redux-form#2088]
IN SHORT: Set focus each time to invalid field on each form sumbit

DESC: I need to set focus on submit to apropriate invalid field. I have all necassary actions to do that and refs for each field. The idea is to fire "setFocus" event from "submitFailed" callback and track onFocus props change in "componentDidUpdate". But problem is that validation on submit fires only once, after first sumbit (which works normally) fields must be "changed" to make the form submitable again, otherwise form won't submit, it just emits  "TOUCH" and "redux-form/SET_SUBMIT_FAILED" event.  I tried to fire off 'redux-form/CHANGE' event, but it does't help. The desired behavior can be achieved only via "INITIALIZE" on each submit or setting "enableReinitialize" and "initialValues" settings. But I really do not want to copy values to another state slice before submit and and reset to that values via"enableReinitialize". I want just submit to fire each time user push the button. I cannot find explanations for this weird behaviour.

**Comment by alex-shamshurin** [comment]
I see a condition in handleSubmit source code. It's `if (valid || persistentSubmitErrors) {...}`. I do not know how to set it valid (may be make sync validation) but I set `persistentSubmitErrors: true` in order to make submit works.

**Comment by gustavohenke** [comment]
Because this issue saw almost no activity for more than a year, I'm closing it.

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (107/100 (107%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from o4.sgmail.github.com ([192.254.112.99]:46121)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.89)
	(envelope-from <bounces+848413-b37a-greg=biomassiv.es@sgmail.github.com>)
	id 1eMimJ-002OCu-SA
	for greg@biomassiv.es; Wed, 06 Dec 2017 17:06:59 -0600
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=github.com; 
	h=from:reply-to:to:cc:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=yWs22DE0Jtrn4ogaDw+WLzD46X0=; b=scKBDrfSjmcOjgVA
	yQCiyyBvUF2YdZAa0rTJ/8dXI6bEe9p5PkpTjreEQe/DpjdFZOLSU2tiCzXj8Ugi
	3WFmW9b3KLrNlmN6VKAdBN4nlx35IU7KG+GA2PjJ7q5jGFZP1sd5HWa2tz2mhz8a
	f9wTuv7Dr+yohA+M8B0tepDs11g=
Received: by filter0436p1iad2.sendgrid.net with SMTP id filter0436p1iad2-1406-5A2877EC-12
        2017-12-06 23:06:20.58292043 +0000 UTC
Received: from github-smtp2b-ext-cp1-prd.iad.github.net (github-smtp2b-ext-cp1-prd.iad.github.net [192.30.253.17])
	by ismtpd0002p1iad1.sendgrid.net (SG) with ESMTP id M7XYLTjtRvS36IxIJ81ERA
	for <greg@biomassiv.es>; Wed, 06 Dec 2017 23:06:20.465 +0000 (UTC)
Date: Wed, 06 Dec 2017 23:06:20 +0000 (UTC)

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (106/100 (106%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext6.iad.github.net ([192.30.252.197]:52002 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.89)
	(envelope-from <noreply@github.com>)
	id 1eMimH-002OC0-WC
	for greg@biomassiv.es; Wed, 06 Dec 2017 17:06:58 -0600
Date: Wed, 06 Dec 2017 15:06:18 -0800
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1512601578;
	bh=3dPmbUmFpga4OnDfl+Y37XSKgCG0Cx+Bc9CqgSIYwdI=;
	h=From:Reply-To:To:Cc:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=o/xUcn9KOSl1POfqNvMcEulZYxBOAQqAjNd6g6wYvzbbZdbRGa2Zr+aOM59JtAwnu
	 ef+Ly5BpSDf6fHw9YwVKPMVzAcRliWXOxBr0ld2Tq9uMgSN17wCEN2rUeSqeANHI7B
	 hcg3yHJummvRHM4lF6znc70BSvgRQhYFPenp1znA=
